specify/specify7
# Display only record sets for the collection

Fixes #4520

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

go to record sets dialog overlay 
verify that all record sets proposed are part of the collection you are connected in
